README.md

```diff
@@ -315,8 +348,20 @@ To have the automod check your rules against a specific comment, run:
 
 ### Reanalyze all posts since a point in time
 
-You can start a job to reanalyse all posts since a certain time. This might be helpful if you added a new rule, and want to run this over posts already checked before the rule was in place. To do this, run:  
-`docker exec -it lemmy_automod_1 bin/console app:trigger App\\Message\\ReanalyzePostsMessage --arg [datetime]`  
+You can start a job to reanalyse all posts since a certain time (including checking image hashes). This might be helpful if you added a new rule, and want to run this over posts already checked before the rule was in place. To do this, run:  
+`docker exec -it lemmy_automod_1 bin/console app:trigger App\\Message\\ReanalyzePostsMessage --arg [datetime]`
 
 For example:  
-`docker exec -it lemmy_automod_1 bin/console app:trigger App\\Message\\ReanalyzePostsMessage --arg '2024-02-16T01:00:00+02:00'`
\ No newline at end of file
+`docker exec -it lemmy_automod_1 bin/console app:trigger App\\Message\\ReanalyzePostsMessage --arg '2024-02-16T01:00:00+02:00'`
+
+### Unban user and restore content
+
+If you have accidentally banned a user, you can unban them using the website UI, but make sure you first change the rule that banned them otherwise the user may be re-banned. If you've removed their content, you can use the following command to unban and restore content:
+
+`docker exec -it lemmy_automod_1 bin/console app:trigger App\\Message\\UnbanUserMessage --arg [username] --arg [instance]`
+
+For example:
+
+`docker exec -it lemmy_automod_1 bin/console app:trigger App\\Message\\UnbanUserMessage --arg 'MyUsername' --arg 'lemmings.world'`
+
+This will unban them. It will also iterate through all posts and comments and make all their commands and posts visible again. Note this will restore all posts and comments, even if the Automod was not the one that removed them, including posts and comments that the user themselves deleted. So be careful using it.
```

Comment:
```suggestion
This will unban them. It will also iterate through all posts and comments and make all their comments and posts visible again. Note this will restore all posts and comments, even if the Automod was not the one that removed them, including posts and comments that the user themselves deleted. So be careful using it.
```